exercises/hardware_access_control/part_2/gpio_access.cc

```diff
@@ -66,7 +66,8 @@ auto acquire_led(uint8_t index) -> std::optional<SealedLedHandle>
 /**
  * Unseal a handle with our led token key.
  */
-static auto unseal_handle(SealedLedHandle handle) -> std::optional<LedHandle const *>
+static auto unseal_handle(SealedLedHandle handle)
+  -> std::optional<LedHandle const *>
```

Comment:
This change and its commit message don't make sense to me. The commit message says it fixes the sense hat demo but the change is an extra line return in hardware access control part 2.

Reply:
Good spot, bad squash, fixed now